### DISCUSSION TITLE
/.ui  :: My Hyper-Blackhole

### DISCUSSION THREAD

**Discussion by theFuzzyWarble** [beakerbrowser/beaker#1851] [Q&A]
@pfrazee While working on figuring out how to create a Frontend UI, I manually created /.ui folder on my drive.
Once I got some pieces in place and working I decided to move the frontend to its own drive.

So I created the new hyperdrive, copied the files over. After deleting the original manual created folder.
I mounted the new drive to /.ui. Worked as expected!

Then I started moving stuff around again, ended up unmounting the /.ui folder.
Now I'm trying to remount it, but it seems /.ui isn't taking my calls anymore!
I can't recreate it as a mount. But I can manually create a /.ui folder. And deleting that  still doesn't let me create a mount to replace it.
I then mounted the same drive to  /.abc and that works just fine.
Trying to be funny but not, I did a  "mv /.abc /.ui" and poof /.abc vanished. and sadly no /.ui either.

And yes, I can recreate the /.abc mount with no problem.

IDEAS?!!!
I hope so, cause I'm about to unload with specific /.ui problems next!!

**Comment by theFuzzyWarble** [comment]
Just a quick update if anyone follows this.
@pfrazee got back to me in irc and this isn't something currently addressable, understandably, but he's noted it.
My hope is it's just a me problem/fluke so I'll switch back to local .UI folder rather than a mount point until then.

[update]: Well, I was wrong about the manually created .ui folder, I can create and remove it but it's not usable.

**Reply by theFuzzyWarble** [reply]
1.0 didn't magically fix this :|
I didn't expect it would but 'tis this season for wishes!!

**Reply by pfrazee** [reply]
Yeah :( this sounds like some really unfortunate bug with the internal state, either Beaker doing something poorly or a bug in the data structure. I'm sorry I don't have the time right now to dig into this
